core/src/test/java/org/apache/calcite/plan/RelWriterTest.java

```diff
@@ -146,8 +147,9 @@
       + "        {\n"
       + "          \"agg\": {\n"
       + "            \"name\": \"COUNT\",\n"
-      + "            \"kind\": \"COUNT\",\n"
-      + "            \"syntax\": \"FUNCTION_STAR\"\n"
+      + "            \"syntax\": \"FUNCTION_STAR\",\n"
+      + "            \"class\": \"org.apache.calcite.sql.fun.SqlCountAggFunction\",\n"
+      + "            \"kind\": \"COUNT\"\n"
```

Comment:
I don't think class info is needed for system aggregate function

Reply:
You are right. Code has been updated.